tests/resources/altitude.json

```diff
@@ -25,7 +25,7 @@
                     "MODEL": {
                         "NAME": "altitude",
                         "VERSION": "0.1.0",
-                        "URL": "https://raw.githubusercontent.com/gammasim/simulation_model/main/schema/altitude.schema.yml"
+                        "URL": "https://raw.githubusercontent.com/gammasim/simulation_model/site-parameters/schema/altitude.schema.yml"
```

Comment:
Should you add a comment somewhere to remember to change it to main later? I know you can't add a comment in json, so maybe an issue would help.

Reply:
Done